packages/sdk-ts/src/client/indexer/grpc/IndexerGrpcDerivativesApi.ts

```diff
@@ -469,8 +469,9 @@ export class IndexerGrpcDerivativesApi extends BaseConsumer {
         DerivativeSubaccountTradesListResponse,
         typeof InjectiveDerivativeExchangeRPC.SubaccountTradesList
       >(request, InjectiveDerivativeExchangeRPC.SubaccountTradesList)
-
-      return IndexerGrpcDerivativeTransformer.tradesResponseToTrades(response)
+      return IndexerGrpcDerivativeTransformer.tradesResponseToTrades(
+        response as DerivativeTradesResponse,
```

Comment:
@bangjelkoski I was getting typing error for the `response` parameter stating that it needs `response` as type `DerivativeTradesResponse`. curious what you think here. i'm also curious if this has anything to do with the dex failing earlier on `DerivativeTradesRequest and SpotTradesRequest`